tests/dotnet-sos/Makefile

```diff
@@ -25,21 +25,22 @@ ext2fs: appdir
 oelldb:
 	git clone --branch working https://github.com/vtikoo/oelldb
 
-dotnet-sos:
-	sudo apt install -y dotnet-sdk-5.0 && \
-	dotnet tool install -g dotnet-sos \
-	&& $(HOME)/.dotnet/tools/dotnet-sos install || true
-
 clean:
 	sudo rm -fr appdir ext2fs rootfs roothash oelldb stdouterr.txt /tmp/myst*
```

Comment:
This is fails on my box when trying to remove /tmp/myst*. It says it is busy. It is probably okay to leave /tmp/myst* files in place. They are removed by the pipeline first when it starts to run.

Reply:
Fixed by not removing /tmp/myst* as part  of `clean` target.